pkg/storage/filesystem/btrfs.go

```diff
@@ -81,6 +81,7 @@ func (b *btrfs) Create(ctx context.Context, name string, policy pkg.RaidProfile,
 		"-L", name,
 		"-d", string(policy),
 		"-m", string(policy),
+		"-f",
```

Comment:
I would really want to not have this flag so deep here by default.
This change the behavior of all the call to `btrfs.Create` while we only need this flag during intialization when we claim the free disks.

Instead I would create a new method CreateForce or something, so its very explicit that the method is destructive. And we use this method only in the initialization code.

Reply:
:+1: